macros/edr/dbt_artifacts/upload_dbt_models.sql

```diff
@@ -71,8 +77,9 @@
         'package_name': node_dict.get('package_name'),
         'original_path': node_dict.get('original_file_path'),
         'path': node_dict.get('path'),
-        'generated_at': elementary.datetime_now_utc_as_string()
-    }%}
+        'generated_at': elementary.datetime_now_utc_as_string(),
+        'bigquery_partition_by': config_dict.get("partition_by"),
```

Comment:
just a thought, maybe we can stick with the `dbt` way of doing double underscore between adapter name and name, (`bigquery__partition_by`)

Reply:
Because for BigQuery users, I'll add BigQuery fields and this `var` _does not_ affect it.
It does affect if I'm a BigQuery user and whether it'll add Redshift fields.